src/components/20-molecules/file-upload/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.0.0
+
+- New property `accessOriginalFile` to choose between compromised and original files. #2036
+- BREAKING CHANGE: Make `showFileOverview` private.
```

Comment:
"Remove property .... for public usage."

Reply:
changed it to: `- BREAKING CHANGE: Remove public property `showFileOverview` and only use it internally.`

Reply:
Who should use that internally? The user of our component? I think you are trying to say that this prop is comp-internal now?! Our users should not now that detail. They only need to know that this prop is not available anymore.

Reply:
Now it is `- BREAKING CHANGE: Remove property 'showFileOverview'.`